cli/internal/oauth/util/util.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"errors"
+	"os/exec"
+	"runtime"
+)
+
+// OpenBrowser opens the specified URL in a browser based on OS.
+func OpenBrowser(url string) (err error) {
+	switch runtime.GOOS {
+	case "linux":
+		err = exec.Command("xdg-open", url).Start()
```

Comment:
I'm not too concerned about this because this is a nice to have, and there's always the fallback of the user opening it directly, so I'd rather not add more dependencies just for this.

I'll take a look at how it's done there and see if it's something we can add without a lot more vendored code though.

Reply:
There's no need to vendor more code for linux, it's just a for loop trying different commands. Windows might require vendoring.